tensorflow/python/layers/convolutional.py

```diff
@@ -1200,16 +1201,238 @@ def conv2d_transpose(inputs,
   return layer.apply(inputs)
 
 
+class Conv3DTranspose(Conv3D):
+  """Transposed 3D convolution layer (sometimes called Deconvolution).
+
+  The need for transposed convolutions generally arises
+  from the desire to use a transformation going in the opposite direction
+  of a normal convolution, i.e., from something that has the shape of the
+  output of some convolution to something that has the shape of its input
+  while maintaining a connectivity pattern that is compatible with
+  said convolution.
```

Comment:
This seems redundant with the docstring for `Conv2DTranspose`. No need to have it twice.